simphony/cuds/simulation.py

```diff
@@ -0,0 +1,28 @@
+"""Contains simulation controllers."""
+from . import CUDS
```

Comment:
Having `.` import it from `__init__` module which I think is slightly better, but I have no strong opinion about it.